MartinP460/react-denmark-map
# Improve styling and SEO parameters in demo

### Proposed changes

Slightly modified the styling of the demo and improved the SEO friendliness of the site by adding more metadata tags and a sitemap. Also added a favicon.

### How to test

1. Run the demo and verify that you can see the favicon.
2. Verify that all buttons have active states.
